### DISCUSSION TITLE
I am planing to make the table responsive with the - string length.

### DISCUSSION THREAD

**Discussion by 4-1-1** [Milkdown/milkdown#572] [Ideas]
```typescipt 
|                               |         |  |  |
| :-------------------------------- | :-------- | :- | :- |
|                                |        |  |  |
| where is the  good pe | wddddd |  |   |
```

Make table column width up to `---------------` length , like percent of the total length, and other column just render minWidth 
`clientWidth/(2*columns.length)`

**Comment by Saul-Mirone** [comment]
I sugguest you to release it as your own plugin.

**Reply by 4-1-1** [reply]
when you input something in a table , the length of '-' will changed, I think It should mean something .
Do you have some plan with table , I just found you take out the depency of 'pm table',  what I was think about it

**Comment by 4-1-1** [comment]
And I have done a html plugin support font color and background color and br, Do you want me to take a pr with them?

**Reply by Saul-Mirone** [reply]
> And I have done a html plugin support font color and background color and br, Do you want me to take a pr with them?

I think you can keep them and publish them as your own plugins. I don't have a plan to maintain them myself because they're not part of commonmark of gfm.
